github/Team.py

```diff
@@ -234,13 +243,23 @@ def get_repo_permission(self, repo):
         except UnknownObjectException:
             return None
 
+    @deprecated(
+        reason="""
+        Team.set_repo_permission() is deprecated, use
+        Team.update_team_repository() instead.
+        """
+    )
     def set_repo_permission(self, repo, permission):
         """
-        :calls: `PUT /teams/:id/repos/:org/:repo <http://developer.github.com/v3/orgs/teams>`_
+        This API call is deprecated. Use `update_team_repository` instead:
+        https://developer.github.com/v3/teams/#add-or-update-team-repository-legacy
```

Comment:
I think we can ignoring changing the doc string --- or the method to add a blank line, and just add the deprecation message.

Reply:
Thanks. Fixed in https://github.com/PyGithub/PyGithub/pull/1509/commits/8896c88e1dd4b6a7e6ef01b51d1223f21e5362a8